update-dependencies/update-dependencies.ps1

```diff
@@ -0,0 +1,71 @@
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+param(
+    [string[]]$EnvVars=@(),
+    [switch]$Help)
+
+if($Help)
+{
+    Write-Host "Usage: .\update-dependencies.ps1 [Options]"
+    Write-Host ""
+    Write-Host "Summary: Installs the .NET CLI and then compiles and invokes update-dependencies.exe."
+    Write-Host ""
+    Write-Host "Options:"
+    Write-Host "  -EnvVars <'V1=val1','V2=val2'...>  Comma separated list of environment variable name-value pairs"
+    Write-Host "  -Help                              Display this help message"
+    exit 0
+}
+
+$Architecture='x64'
+
+$RepoRoot = "$PSScriptRoot\.."
+$AppPath = "$PSScriptRoot"
+
+# Use a repo-local install directory (but not the artifacts directory because that gets cleaned a lot)
+if (!$env:DOTNET_INSTALL_DIR)
+{
+    $env:DOTNET_INSTALL_DIR="$RepoRoot\.dotnet_stage0\Windows\$Architecture"
```

Comment:
>.dotnet_stage0 [](start = 39, length = 14)

What do you think about naming this something besides stage0?  This IMO is something that makes sense in CLI but not in this context.  I would lean towards simple .dotnet